packages/postgres-database/src/simcore_postgres_database/models/project_to_groups.py

```diff
@@ -0,0 +1,63 @@
+import sqlalchemy as sa
+from sqlalchemy.sql import expression
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+from .groups import groups
+from .projects import projects
+
+project_to_groups = sa.Table(
```

Comment:
Lets agree on how we will do it. Currently we have:
- cluster_to_groups
- tags_to_groups
- user_to_groups
- wallet_to_groups
- new: project_to_groups
with the same structure (resource, gid, read, write, delete, created, modified)

The only place we do not have this naming and structure is:
- services_access_rights
with structure (key, version, gid, execute_access, write_access, created, modified, product)

Also I am trying to be consistent and have the same API everywhere currently implemented in wallet and project resource
- GET, PUT, DELETE `/resource/<id>/groups/<groups-id> `
- example: `/projects/<project-id>/groups/<groups-id>`

I understand that **access_rights** might be more easy to read, but `project_to_groups` defines the relation between 2 resources projects and groups therefore this is also valid name and I would stay consistent everywhere.

Reply:
I still find it confusing. In my mind, `resoruceX_to_resourceY` is used as many to many table with just two primary keys inside. It does not contain important information like `access_rights`. So I do not share your view on this one.
We go with this version you are proposing.

Reply:
Yes after discussion I realized you are right that this is one to many relationships (which we are already trying to hack in the name because if you check the resource is in Singular form `project_to_groups`) therefore resource_access_rights would be a better naming convention. For now consistency argument wins for me, therefore I will leave it like this. We can always rename everything in a separate PR.